test/run-tests.py

```diff
@@ -6,7 +6,7 @@
 passed, failed, verbose, quiet = 0, 0, 0, 0
 
 def run (forthfile):
-    return os.popen ("gforth ../ttester-codewars.4th " + forthfile + " -e bye").readlines ()
+    return os.popen ("gforth ../ttester-codewars.4th " + forthfile + " -e bye 2>/dev/null").readlines ()
```

Comment:
Should the runner remove `stderr` output as well? I'm assuming this is for redefining `->`.
If Forth outputs useful messages on `stderr`, I'd like to avoid removing everything. Is the warning in some format so that we can filter specific warnings?

Reply:
You're right. It is about the redefining warning, which cannot be turned off. I thought that would be acceptable in running the test suite. I think it just is a string 'redefining <word>' in stderr. Is 'sed' allowed? In that case the redirection can be replaced by
`2&>1 | sed -E 's/redefined [^ ]+//g'`

Reply:
Cool, the runner can ignore and filter the captured stderr before responding so I'll check the output and decide what to do.